src/conv/problem_description.cpp

```diff
@@ -187,6 +187,19 @@ void ProblemDescription::MakeNetworkConfig(std::string& conf_key) const
     ss << 'x' << GetDirectionStr();
     ss << 'x' << GetAlphaBetaCaseStr();
 
+    // When alpha = 1.1 and beta = 2.2, kernel is cached as bilinear.
+    // In next run if alpha = 3.3 and beta = 3.3,
+    // same cached bilinear kernel (alpha = 1.1 and beta = 2.2)
+    // was retrived. So, had to add alpha value to
+    // network config to avoid picking wrongly cached kernel.
+    auto roundUpBilinearValues = [](double val) -> std::string {
+        std::ostringstream stream;
+        stream << std::fixed << std::setprecision(1) << val;
+        return stream.str();
+    };
+    ss << 'x' << roundUpBilinearValues(GetAlpha().GetAsDouble());
+    ss << 'x' << roundUpBilinearValues(GetBeta().GetAsDouble());
+
```

Comment:
@amberhassaan  test were failing since wrong cached kernel were used.

Reply:
No, this is not acceptable. We need to find out why the wrong kernel is being picked up. One possible common cause would be that we are caching certain variables in the Invoker and those cached values end up being wrong when the problem changes. We need to find out if that's happening and what those values are. Put prints inside the Invoker for failing tests.

Reply:
what if problem is same but the alpha and beta values are different?